help/dark-theme.md

```diff
@@ -9,10 +9,12 @@ for working in a dark space.
 
 {tab|desktop-web}
 
-{settings_tab|preferences}
+{!personal-menu.md!}
 
-1. Under **General**, select the desired color scheme from the **Theme**
-   dropdown.
+1. Locate the **theme switcher** and select the desired color scheme
+   out of **Automatic (follows system settings)** (<i class="zulip-icon zulip-icon-monitor"></i>),
```

Comment:
I would write:

> Select the desired theme using the row of icons in the middle of the menu.

I came up with that by referring to https://zulip.com/help/follow-a-topic, which has a similar UI.

Reply:
Updated that line based on your feedback, thanks! Please also check the updated screenshot in PR description.